### PR TITLE
Update wording and severity for the 'Downloading Managed Dependencies message'

### PR DESCRIPTION
**Pull request by Francisco-Gamino**
Addressing https://github.com/Azure/azure-functions-powershell-worker/issues/362

### CODE REVIEW

File: src/resources/PowerShellWorkerStrings.resx [hunk 1]
```diff
@@ -59,8 +59,8 @@
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema id="root" xmlns="" xmlns:xsd="https://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
-    <xsd:import namespace="https://www.w3.org/XML/1998/namespace" />
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```
Comment (AnatoliB) [inline src/resources/PowerShellWorkerStrings.resx:62]: Wondering about the "https" -> "http" change... Was this done by a tool automatically?

Comment (AnatoliB) [reply src/resources/PowerShellWorkerStrings.resx:62]: I see: VS 2019 does that. Ok.

**Review by AnatoliB — APPROVED** [review]
LGTM

File: src/resources/PowerShellWorkerStrings.resx (hunk 1 above)
Comment (Francisco-Gamino) [reply src/resources/PowerShellWorkerStrings.resx:62]: > I see: VS 2019 does that. Ok.

Ok, so I will leave it as is. Thanks for the code review.

**Pull request merged**